src/Azure/Queues/AzureServiceBusQueue.cs

```diff
@@ -54,6 +65,14 @@ public class AzureServiceBusQueue<T> : QueueBase<T> where T : class {
                     changes = true;
                 }
 
+                if (_queueDescription.AutoDeleteOnIdle != _autoDeleteOnIdle) {
+                    _queueDescription.AutoDeleteOnIdle = _autoDeleteOnIdle;
+                }

```

Comment:
@trbenning I think we are missing the changes = true check as in the lines above. Can you please confirm this.